home/rvagg/2021-02-exploration-report-small-filecoin-deals.md

```diff
@@ -0,0 +1,158 @@
+# @rvagg Exploration Report: Small Filecoin Deals (2021-02)
+
+## Background
+
+### Filecoin storage for the uninitiated
+
+* Typical miners are continually "sealing" new sectors, containing either 32 GiB or 64 GiB of data
+* Sealed sectors have a lifetime, pre-specified by the miner, up to a maximum of 18 months and a minimum of 6 months
+* Sealed sectors contribute to miner "power", power increases a miner's chances of winning block rewards
+* Sectors may, or may not contain data from clients in the form of paid "deals"
+* Sectors not containing deals are called "committed capacity" (CC) sectors, they simply contribute to the total network capacity without storing useful data
+   * CC sectors remove the incentive to self-deal (faux-deal) where miners pretend to have deal data to increase power
+   * Total network capacity, including CC, signals to users that the network has real hardware ready to store data even if those sectors currently don't
+* Sectors that contain deal data are called "regular sectors"
+* There is no requirement for a mix of data types in sealed sectors (deal / non-deal)
```

Comment:
I found this hard to parse in my head but I think it means the data type in a sealed sector _can_ be mixed deal/non-deal?

Reply:
Poorly worded, sorry. So this image under "sector quality" in the specs is kind of interesting to illustrate what this is talking about (even though this image is designed to talk about "quality" -> "power", hence the multipliers):

![sector quality](https://spec.filecoin.io/systems/filecoin_mining/sector/sector-quality/sector-quality.jpg)

Basically, a sector can contain no deals, some deals, all deals, and those deals can be regular and verified, varying sizes, in any mix, as long as they fit nicely into the power-of-two offsets.